grafana/resource_data_source_test.go

```diff
@@ -448,21 +448,21 @@ func TestAccDataSource_basic(t *testing.T) {
 			resource: "grafana_data_source.github",
 			config: `
 			resource "grafana_data_source" "github" {
-				type = "github"
+				type = "grafana-github-datasource"
 				name = "github"
 				json_data {
 					github_url = "https://github.com"
 				}
 				secure_json_data {
-					access_token = "token for github"
+					accessToken = "token for github"
```

Comment:
Here also, `access_token` refers to the Terraform format. So it was already correct

Reply:
Understood thanks again. Latest push corrects this.